cli/foundation-models/system/inference/text-to-image/text-to-image-online-endpoint.sh

```diff
@@ -55,14 +55,22 @@ max_concurrent_request=2  # the maximum number of concurrent requests supported
 # If you are using a larger SKU, please increase this value to get the maximum performance.
 
 # Deploy model from registry to endpoint in workspace
+
 az ml online-deployment create --file deploy-online.yaml $workspace_info --all-traffic --set \
```

Comment:
this step runs into " (InvalidAuthenticationTokenTenant) The access token is from the wrong issuer"

 issue